django-server/server/api/models.py

```diff
@@ -124,3 +124,10 @@ def save(self, *args, **kwargs):
     def calculate_new_avg_review(self, old_review_stars, number_of_reviews):
         current_sum = old_review_stars * number_of_reviews + self.stars
         return current_sum / (number_of_reviews + 1)
+
+
+class OccupancyStatsData(object):
+
+    def __init__(self, **kwargs):
+        for field in ('id', 'timestamp_name', 'occupancy', 'occupancy_relative'):
```

Comment:
Was it possible to use custom object for occupancy? This is low priority here.

Reply:
I don't understand the question here